fenapack/field_split_backend.py

```diff
@@ -144,6 +148,14 @@ def setup_ksp_once(self, ksp, assemble_func, iset, spd=False):
             # Setup ksp
             with Timer("FENaPack: {} setup".format(prefix)):
                 ksp.setUp()
+        elif not const:
+            # Reinitialize matrix and ksp
```

Comment:
Reinitialize? Assembly would be sufficent, wouldn't it?

Reply:
hairsplitting :-)